schematic/json_schema_functions.py

```diff
@@ -0,0 +1,368 @@
+
+    Returns:
+        The URI for the JSON Schema
+    """
+
+    js_service = syn.service("json_schema")
+    org = js_service.JsonSchemaOrganization(synapse_org_name)
+    org.create_json_schema(json_schema, schema_name, schema_version)
+    uri = f"{synapse_org_name}-{schema_name}-{schema_version}"
+    return uri
+
+
+@deprecated(reason="Entity view functionality is only need temporarily")
+def create_json_schema_entity_view(
+    syn: Synapse,
+    entity_id: str,
+    parent_id: str,
+    entity_view_name: str = "JSON Schema view",
+) -> str:
+    """
+    Creates A Synapse entity view based on a JSON Schema that is bound to a Synapse entity
+    This functionality is needed only temporarily. See note at top of module.
+
+    Args:
+        syn: A Synapse object thats been logged in
+        entity_id: The ID of the entity in Synapse to bind the JSON Schema to
+        parent_id: The ID of the entity in Synapse to put the entity view and wiki at
+        entity_view_name: The name the crated entity view will have
+
+    Returns:
+        The Synapse id of the crated entity view
+    """
+    syn.get_available_services()
+    js_service = syn.service("json_schema")
+    json_schema = js_service.get_json_schema(entity_id)
+    my_org = js_service.JsonSchemaOrganization(
+        json_schema["jsonSchemaVersionInfo"]["organizationName"]
+    )
+    schema_version = js_service.JsonSchemaVersion(
+        my_org,
+        json_schema["jsonSchemaVersionInfo"]["schemaName"],
+        json_schema["jsonSchemaVersionInfo"]["semanticVersion"],
+    )
+    columns = _create_columns_from_json_schema(schema_version.body)
+    view = EntityView(
+        name=entity_view_name,
+        parent_id=parent_id,
+        scope_ids=[entity_id],
+        view_type_mask=ViewTypeMask.FILE,
+        columns=columns,
+    ).store(synapse_client=syn)
+    view.reorder_column(name="createdBy", index=0)
+    view.reorder_column(name="name", index=0)
+    view.reorder_column(name="id", index=0)
+    view.store(synapse_client=syn)
+    return view.id
+
+
+@deprecated(reason="Entity view functionality is only need temporarily")
+def create_entity_view_wiki(
+    syn: Synapse, entity_view_id: str, owner_id: str, title: str
+) -> Wiki:
+    """
+    Creates a wiki for a entity view in Synapse
+    This functionality is needed only temporarily. See note at top of module.
+
+    Args:
+        syn: A Synapse object thats been logged in
+        entity_view_id: The Synapse id of the entity view to make the wiki for
+        owner_id: The ID of the entity in Synapse to put as owner of the wiki
+        title: The title of the wiki to be created
+
+    Returns:
+        The created wiki object
+    """
+    content = (
+        "${synapsetable?query=select %2A from "
+        f"{entity_view_id}"
+        "&showquery=false&tableonly=false}"
+    )
+    wiki = Wiki(title=title, owner=owner_id, markdown=content)
```

Comment:
Is there a way to append this content to the end of the wiki, in case there is already content?

Reply:
This will now create/add to the wiki content depending on if the wiki exists or not.